infrastructure/storageprovider/dropbox.go

```diff
@@ -122,3 +148,29 @@ func (d *dropbox) prepareRequest(accessToken string, file io.Reader, fileName, s
 	req.Header.Set("Dropbox-API-Arg", dropboxAPIArg)
 	return req, nil
 }
+
+func (d *dropbox) mapToDropboxError(byteResponse *[]byte, httpStatusCode int) dropboxError {
+	strRes := string(*byteResponse)
+	errorRes := dropboxError{HttpCode: httpStatusCode}
+	errorRes.Message = strRes
```

Comment:
is ``strRes`` variable reusable? 
I think it should be changed into ``errorRes.Message = string(*byteResponse)``

Reply:
Okay